src/batch-client/index.ts

```diff
@@ -1687,6 +1687,23 @@ export class ZimbraBatchClient {
 			accountName: accountName
 		}).then(normalize(SendMessageInfo));
 
+	public sendSecureMessage = (
+		message: SendMessageInput,
+		accountName: string,
+		sign: Boolean,
+		encrypt: Boolean
+	) =>
+		this.jsonRequest({
+			name: 'SendSecureMsg',
```

Comment:
we can go one more level to reduce code duplication, update sendMessage function to change api name based on presence of sing/encrypt flags

Reply:
Fixed.